frontend/packages/console-app/src/components/nodes/NodesPage.tsx

```diff
@@ -622,6 +642,23 @@ const getFilters = (t: TFunction): RowFilter<NodeRowItem>[] => [
       return input.selected?.some((r) => nodeRoles.includes(r));
     },
   },
+  {
+    filterGroupName: t('console-app~Architecture'),
+    type: 'cpu-arch',
+    reducer: getNodeArchitecture,
+    items: [
+      { id: 'amd64', title: t('amd64') },
+      { id: 'ppc64le', title: t('ppc64le') },
+      { id: 'arm64', title: t('arm64') },
+      { id: 's390x', title: t('s390x') },
```

Comment:
I don't think we need to translate these strings:
```suggestion
      { id: 'amd64', title: 'amd64' },
      { id: 'ppc64le', title: 'ppc64le' },
      { id: 'arm64', title: 'arm64' },
      { id: 's390x', title: 's390x' },
```

Reply:
I don't think it's the case that every Node has an architecture at the time it's added to the cluster. If I recall correctly in some UPI cases, it can be added after the node joins the cluster.